thirdparty/github.go

```diff
@@ -267,8 +263,20 @@ func GetInstallationToken(ctx context.Context, owner, repo string, opts *github.
 	return token, nil
 }
 
-func getInstallationTokenWithoutOwnerRepo(ctx context.Context) (string, error) {
-	return GetInstallationToken(ctx, defaultOwner, defaultRepo, nil)
+func getInstallationTokenWithDefaultOwnerRepo(ctx context.Context, opts *github.InstallationTokenOptions) (string, error) {
+	settings, err := evergreen.GetConfig()
+	if err != nil {
+		return "", errors.Wrap(err, "getting evergreen settings")
+	}
+	token, err := settings.CreateInstallationTokenWithDefaultOwnerRepo(ctx, opts)
```

Comment:
Can you explain why we need a default owner/repo? Do we intend to keep this? What are we setting this to?

Reply:
The motivation for this ticket comes from [the conversation described in this comment](https://github.com/evergreen-ci/evergreen/pull/6559#pullrequestreview-1455695124). Adding a comment to the function as to the motivation for when/where to use the default owner/repo func would be helpful.

Reply:
In functions like CheckGithubAPILimit, we dont provide the owner and repo because we don't need it. 
But if we want to authenticate as app, we still need an owner and repo to get the installation token for. 
So this is in that case, we would just want to use the evergreen-ci/evergreen aka the default repo. 
We agreed that it should be configurable on admin settings instead of having it be hard coded like it was previously but I separated this work out of the last pr!